hazelcast/src/main/java/com/hazelcast/config/BaseMetricsConfig.java

```diff
@@ -86,6 +90,10 @@ public MetricsJmxConfig getJmxConfig() {
      * metrics are collected every 5 seconds.
      * <p>
      * May be overridden by {@link ClusterProperty#METRICS_COLLECTION_FREQUENCY}
+     * May be overridden by followings for the client.
+     * {@link com.hazelcast.client.properties.ClientProperty#METRICS_COLLECTION_FREQUENCY}
+     * {@link com.hazelcast.client.properties.ClientProperty#STATISTICS_PERIOD_SECONDS}
+     * When both of them is configured for the client, STATISTICS_PERIOD_SECONDS is ignored.
      * system property.
      */
```

Comment:
```suggestion
     * May be overridden by {@link ClusterProperty#METRICS_COLLECTION_FREQUENCY}
     * system property for the member.
     * <p>
     * May be overridden by {@link ClientProperty#METRICS_COLLECTION_FREQUENCY}
     * and {@link ClientProperty#STATISTICS_PERIOD_SECONDS} system properties
     * for the client.
     * <p>
     * When both of them are configured for the client,
     * {@link ClientProperty#STATISTICS_PERIOD_SECONDS} is ignored.
     */
```
Added some line breaks, used `ClientProperty` directly to have a shorter text (please add import at the top), and fixed the sentence for the member side overriding, as the `system property.` was moved to the end.